application/reader/services/reader_simulation_service.py

```
@@ -0,0 +1,338 @@
+                    suspense_retention=fb.scores.suspense_retention,
+                    thrill_score=fb.scores.thrill_score,
+                    churn_risk=fb.scores.churn_risk,
+                    emotional_resonance=fb.scores.emotional_resonance,
+                ),
+                one_line_verdict=fb.one_line_verdict,
+                highlights=list(fb.highlights),
+                pain_points=list(fb.pain_points),
+                suggestions=list(fb.suggestions),
+            ))
+
+        # 确保三个人设都有（缺失时填默认）
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)